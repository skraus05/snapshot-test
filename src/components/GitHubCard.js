import { Card } from 'react-bootstrap'

export default function GitHubCard() {
    return (
        <Card>Samuel</Card>
    )
}