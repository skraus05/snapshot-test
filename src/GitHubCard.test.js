import GitHubCard from './components/GitHubCard';
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard/>).toJSON()
  expect(tree).toMatchSnapshot()
});