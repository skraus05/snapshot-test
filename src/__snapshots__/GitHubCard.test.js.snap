// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  Samuel
</div>
`;
